src/lib.rs

```diff
@@ -426,28 +426,29 @@ impl<'a, 'mir, 'tcx> Machine<'a, 'mir, 'tcx> for Evaluator<'tcx> {
             malloc_mir.span,
             malloc_mir,
             Some(dest),
-            // Don't do anything when we are done.  The statement() function will increment
+            // Don't do anything when we are done. The `statement()` function will increment
             // the old stack frame's stmt counter to the next statement, which means that when
-            // exchange_malloc returns, we go on evaluating exactly where we want to be.
+            // `exchange_malloc` returns, we go on evaluating exactly where we want to be.
             StackPopCleanup::None { cleanup: true },
         )?;
 
         let mut args = ecx.frame().mir.args_iter();
         let layout = ecx.layout_of(dest.layout.ty.builtin_deref(false).unwrap().ty)?;
 
-        // First argument: size
-        // (0 is allowed here, this is expected to be handled by the lang item)
+        // First argument: `size`.
+        // (`0` is allowed here -- this is expected to be handled by the lang item).
         let arg = ecx.eval_place(&mir::Place::Local(args.next().unwrap()))?;
         let size = layout.size.bytes();
         ecx.write_scalar(Scalar::from_uint(size, arg.layout.size), arg)?;
 
-        // Second argument: align
+        // Second argument: `align`.
         let arg = ecx.eval_place(&mir::Place::Local(args.next().unwrap()))?;
         let align = layout.align.abi.bytes();
         ecx.write_scalar(Scalar::from_uint(align, arg.layout.size), arg)?;
 
-        // No more arguments
-        assert!(args.next().is_none(), "exchange_malloc lang item has more arguments than expected");
+        // No more arguments.
+        assert!(args.next().is_none(), "`exchange_malloc` lang item has more arguments \
```

Comment:
keep the old code

Reply:
This is convention in Rustc. Let's follow it here too, no? :-)

Reply:
if anything, rewrite it in block indent, otherwise keep the original